### PR TITLE
XML Binding 4.0: Marshalling issue fix

### LINKED ISSUE #26144: XML Binding 4.0: Marshalling result mismatch on Windows platform
Result of  XML Binding  marshalling on windows platform for EE 10, generates not identical results than other platforms. Even though XML structure is same, additional whitespace tabs or this kind of non visible characters were causing a mismatch.

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #26144

### CODE REVIEW

**Comment by BerksanAtes** [comment]
SOE PB -> https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_ao2mEEjmEe6Dy7HKT-o-zA

SOE PB with duplicate element prevention code -> https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_4vIWsEwTEe6Ir8wJScnZSQ

File: dev/io.openliberty.xmlBinding.4.0.internal_fat/test-applications/thirdPartyJaxbImplContextApp/src/jaxbimpl/thirdparty/web/ThirdPartyJAXBImplContextTestServlet.java [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /*******************************************************************************
- * Copyright (c) 2022 IBM Corporation and others.
+ * Copyright (c) 2022, 2023 IBM Corporation and others.
```
Comment (neuwerk) [inline dev/io.openliberty.xmlBinding.4.0.internal_fat/test-applications/thirdPartyJaxbImplContextApp/src/jaxbimpl/thirdparty/web/ThirdPartyJAXBImplContextTestServlet.java:2]: Can you please add a comment to add as a reminder that if this test class continues to have failures related to unmarshalling/marshalling, that we should rewrite the tests to remove the direct comparison on serialized results?

**Review by neuwerk — COMMENTED** [review]
Looks great over all, just see my review comment.

**Review by neuwerk — APPROVED** [review]
LGTM!

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**